src/AddTodoForm.jsx

```diff
@@ -0,0 +1,11 @@
+function AddTodoForm() {
+  return (
+    <form>
+      <label htmlFor="todoTitle">Title </label>
+      <input id="todoTitle" type="text" />
+      <button>Add</button>
```

Comment:
Adding type="submit" to the button is a good practice to define its behavior in the form.

Reply:
Hi Anna, I changed it

Reply:
Well done!